packages/jest-message-util/src/index.ts

```diff
@@ -383,15 +391,47 @@ export const formatResultsErrors = (
   }
 
   return failedResults
-    .map(({result, content}) => {
-      let {message, stack} = separateMessageFromStack(content);
-      stack = options.noStackTrace
-        ? ''
-        : `${STACK_TRACE_COLOR(
-            formatStackTrace(stack, config, options, testPath),
-          )}\n`;
+    .map(({result, content, rawError}) => {
+      function formatErrorStack(errorOrStack: Error | string): string {
```

Comment:
why is this function defined inline?

Reply:
I'll extract it as a standalone function

Reply:
Extracted as a standalone function